pkgs/data/fonts/fira-code/symbols.nix

```diff
@@ -1,26 +1,36 @@
-{ stdenv, fetchzip }:
+{ stdenv, fira-code, python3, fontforge }:
 
-fetchzip {
-  name = "fira-code-symbols-20160811";
+let pythonEnv = python3.withPackages (p: [ p.fontforge ]);
+in stdenv.mkDerivation {
+  name = "fira-code-symbols-20191205";
+  # https://gist.github.com/xieve/d5a01cc59896c3973cb16df9ba8d30d4
+  src = ./fira_code_patch.py;
 
-  url = "https://github.com/tonsky/FiraCode/files/412440/FiraCode-Regular-Symbol.zip";
+  nativeBuildInputs = [ fontforge pythonEnv fira-code ];
 
-  postFetch = ''
-    mkdir -p $out/share/fonts
-    unzip -j $downloadedFile -d $out/share/fonts/opentype
-  '';
+  phases = [ "buildPhase" "installPhase" ];
 
-  sha256 = "19krsp22rin74ix0i19v4bh1c965g18xkmz1n55h6n6qimisnbkm";
+  buildPhase = ''
+    python3 $src -o . ${fira-code}/share/fonts/opentype/*.otf
```

Comment:
no need to specify python3, I'd just do either ``python`` or ``${python.interpreter}``